apps/dashboard/src/components/product-pages/common/GameCard.tsx

```diff
@@ -70,7 +70,7 @@ export const GameCard: React.FC<GameCardProps> = ({
               (max-width: 1200px) 50vw,
               33vw"
         />
-        <Center position="absolute" top={0} bottom={0} left={0} right={0}>
+        <div className="flex items-center justify-center absolute top-0 bottom-0 left-0 right-0">
```

Comment:
you can use `inset-0` instead to set all 4 positions

Reply:
til tbh

Reply:
nicee